src/locale/id.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
   'mongoId': '<%= propertyName %> tidak valid.',
   'multipleOf:$1': '<%= propertyName %> harus merupakan kelipatan <%= ruleParams[0] %>.',
   'nonBlank': 'Input ini wajib diisi.',
+  'notDisposableEmail': 'Domain tidak valid.',
```

Comment:
Shouldn't this "Email tidak valid" or something?

Reply:
Nvm turns out it needs `email` validation, also it's easier for user to differentiate the validation error